docker/nginx/entrypoint.sh

```diff
@@ -13,6 +13,8 @@ envsubst '
     ${MF_USERS_HTTP_PORT}
     ${MF_THINGS_HTTP_PORT}
     ${MF_HTTP_ADAPTER_PORT}
+    ${MF_NGINX_MQTT_PORT}
```

Comment:
Why `MF_NGINX_MQTT_PORT`?

Reply:
@manuio what would you suggest?

Reply:
@drasko something without `NGINX`. Maybe `MF_MQTT_BROKER_PORT`

Reply:
ok, whatever you find suitable, but this is MQTT port on NGINX, and there is MQTT port on mqtt-adapter and MQTT port on mqtt broker, so I think it is not bad to denote this in naming

Reply:
ok sorry, my bad